string agent_id
string destination
bool authorized
string path
